saved-reports.md

```diff
@@ -42,7 +42,7 @@ The following fields apply to each map item under the `reports` key:
   * `service`
   * `statefulset`
   * `team`
-* `idle` idle cost allocation, supports `hide`, `share`, and `separate`
+* `idle` idle cost allocation, supports `hide`, `shareByNode`, `shareByCluster` and `separate`
```

Comment:
Not an Oxford comma guy, I see... :eyes:

Reply:
My Oxford comma stance has oscillated between pro- and con- many times over the years. I can't even remember what I actually believe anymore